representation/lsk.metta

```diff
@@ -9,21 +9,21 @@
 ;; Returns:
 ;;  $tree 
 ;;  A LogicalSubtreeKnob (LSK) configured based on the subtree's presence or its appended state.
-(: logicalSubtreeKnob (-> (Tree $a) NodeId (Tree $a) (Tree $a LogicalSubtreeKnob)))
+;; (:logicalSubtreeKnob (-> (Tree $a) NodeId (Tree $a) (Tree $a LogicalSubtreeKnob)))
 (= (logicalSubtreeKnob $tree (mkNodeId $target) $subtree)
    (let*
    (
     ($negatedSubtree (insertAbove $subtree (mkNode NOT)))
-    ; (() (println! (NegatedSubtree : $negatedSubtree)))
-    ($reducedSubtree (cleanTree $negatedSubtree))  ;;;after the reduct integration, the reduction step will be applied here 
-    ; (() (println! (ReducedSubtree: $reducedSubtree))) 
-    ($childrenOfTarget (getChildrenById $tree (mkNodeId $target)))
+    ; ($_ (println! (NegatedSubtree : $negatedSubtree)))
+    ($reducedSubtree  $negatedSubtree) ;; (cleanTree $negatedSubtree))  FIX uncomment after the reduct refactor to petta 
+    ; ($_ (println! (ReducedSubtree: $reducedSubtree))) 
+    ($childrenOfTarget (eval (getChildrenById $tree (mkNodeId $target))))
    )
-   (if (List.contains $subtree $childrenOfTarget) 
+   (if (== True (List.contains $subtree $childrenOfTarget) )
```

Comment:
Are you sure that we don't need to add `eval` here? If the `List.contains` doesn't reduce, we will be getting `False` everytime